### ISSUE TITLE
Action creators within enqueueActions causing a lint error false positive

### ISSUE THREAD

**Issue by rjchow** [rlaffers/eslint-plugin-xstate#29]
**Describe the bug**

This should not cause a lint error as per the [V5 update guidance](https://stately.ai/docs/migration#use-enqueueactions-instead-of-pure-and-choose): 
```
enqueueActions( ( { enqueue } ) => {
    enqueue( assign() );
}
```

**Versions (please complete the following information):**
 - Node version: 20
 - ESLint version: 8
 - eslint-plugin-xstate version: 3.2.1

**Additional context**
Just need to update https://github.com/rlaffers/eslint-plugin-xstate/blob/9988c15a5c739779837687d752b477d8fa9764b8/lib/rules/no-imperative-action.js#L13
